packages/sakuli-legacy/src/context/selenium-config/safari-properties.class.ts

```diff
@@ -0,0 +1,10 @@
+import {Maybe, Property} from "@sakuli/commons";
+import {logging} from "selenium-webdriver";
+import Preferences = logging.Preferences;
+
+export class SafariProperties {
```

Comment:
The corresponding `safariOptionsFactory` is missing?

Reply:
Yes and no. It turn out, that the @types/selenium-webdriver and the actual selenium-webdriver are out of sync (for many of this option classes). The SafariOption class has no real benefit, therefore i omitted the "safariOptionsFactory". I'll remove the properties class for now.